mimic/canned_responses/fastly.py

```diff
@@ -0,0 +1,278 @@
+"""
+Canned response for fastly
+"""
+import uuid
+
+
+class FastlyResponse(object):
+
+    """
+    Canned response for fastly.
+    See docs here https://docs.fastly.com/api/config
+    """
+
+    fastly_cache = {}
+
+    def get_current_customer(self):
+        """
+        Returns the current customer with response code 200.
+
+        :return: a JSON-serializable dictionary matching the format of the JSON
+                 response for fastly_client.get_current_customer()
+                 ("/current_customer") request.
+        """
+        current_customer = {
+            u'can_edit_matches': u'0',
+            u'can_read_public_ip_list': u'0',
+            u'can_upload_vcl': u'1',
+            u'updated_at': u'2014-11-03T23:37:44+00:00',
+            u'has_config_panel': u'1',
+            u'has_improved_ssl_config': False,
+            u'id': u'3bKRHISECARTtoUMxW6',
+            u'has_historical_stats': u'1',
+            u'has_openstack_logging': u'0',
+            u'can_configure_wordpress': u'0',
+            u'has_improved_logging': u'1',
+            u'readonly': '',
+            u'ip_whitelist': u'0.0.0.0/0',
+            u'owner_id': u'asewQWEwdRnLnCKyMztZqkQvy',
```

Comment:
@glyph, In the context of Openstack Poppy (which is going to be the main/only(?) consumer of Fastly Mimic), there will be only a single owner. Poppy enables provisioning on CDN providers using a single Fastly account. That being said it is really easy for me to update this to generate a unique ID & tht  will still align with Poppy's use of Mimic . Let me know what you think.